packages/mongodb-chatbot-ui/src/messageLinks.tsx

```diff
@@ -42,3 +26,12 @@ export function formatReferences(
     return richLinkProps;
   });
 }
+
+export function getMessageLinks(
```

Comment:
should there be tests for this?

Reply:
Yeah - I actually have some tests for this file in another PR. I'll port them over to this one